frontend/app/views/info/about.scala.html

```diff
@@ -3,6 +3,15 @@
 @import configuration.Config
 @import views.support.Asset
 @import com.gu.membership.salesforce.Tier
+@import org.joda.time.DateTime
+@import com.github.nscala_time.time.Imports._
+
+@* //TODO remove this and relevant imports on 31/3/2015 *@
+@foundingMember(classes: Seq[String] = Nil) = {
+    @if(DateTime.now < new DateTime(2015, 3, 31, 0, 0).withZone(DateTimeZone.UTC)) {
```

Comment:
Keep it simple! Remove the time condition, and we will deploy this on the morning of Tuesday 31st March - we'll set a calendar entry to remind us.